merlin_models/tf/__init__.py

```diff
@@ -85,6 +85,16 @@
     NextItemPredictionTask,
     YoutubeDNNRetrieval,
 )
+from .prediction.losses import (
+    AdaptiveHinge_Loss,
```

Comment:
Our public API might get too cluttered when we put all of this here. Maybe we should have the import be: `merlin_models.loss.HingLoss`. Also I think we should remove the `_` since in python it’s camel-case for classes and snake-case for functions & variables.

Reply:
Done